common/lib/xmodule/xmodule/mongo_connection.py

```diff
@@ -0,0 +1,53 @@
+"""
+Common MongoDB connection functions.
+"""
+import pymongo
+from mongodb_proxy import MongoProxy
+
+
+# pylint: disable=bad-continuation
+def connect_to_mongodb(
+    db, host,
+    port=27017, tz_aware=True, user=None, password=None,
+    retry_wait_time=0.1, proxy=True, try_secondary_read=False, **kwargs
```

Comment:
Could we change the name of 'try_secondary_read' to 'secondaryPreferred' to match pymongo/mongodb's language?